app/javascript/styles/components/CommunityProfiles.scss

```diff
@@ -40,6 +40,7 @@
   }
   header.print-header {
```

Comment:
Would this rule also work if we used only the class selector `.print-header`?

Reply:
This is the only place where that class is used, so I reckon. I'll remove the `header` bit.